repos/system_upgrade/common/models/packagemanagerinfo.py

```diff
@@ -17,3 +17,10 @@ class PkgManagerInfo(Model):
     In case the value is empty string, it means the file exists but it is empty. In such a case the
     original configuration is obviously broken.
     """
+
+    configured_proxies = fields.List(fields.String(), default=[])
+    """
+    Contain a set of proxies from configuration files of yum and dnf.
```

Comment:
If you decide to sort the proxies as suggested above.
```suggestion
    A sorted list of proxies present in yum and dnf configuration files.
```
If you are not going to sort it then:
```suggestion
    A list of proxies present in yum and dnf configuration files.
```